tests/Integration/Monitoring/Test_MonitoringReporter.py

```diff
@@ -250,6 +251,12 @@ def test_addComponentRecords(self):
     self.assertTrue(result['OK'])
     self.assertEqual(result['Value'], len(self.data))
 
+  def test_addComponentsActivityRecords(self):
```

Comment:
You should add different data. self.data is containing WMSMonitoring informations.
Please create a new dictionary. In addition it is probably better to create  a new test (TestComponentsActivityMonitoringTestCase), because you will have to add more test...